packages/xod-client-electron/src/view/containers/App.jsx

```diff
@@ -179,6 +179,7 @@ class App extends client.App {
     const errored = foldEither(
       (error) => {
         proc.fail(error.message, 0);
+        this.props.actions.updateCompileLimit();
```

Comment:
I'm not sure that we really need to `updateCompileLimit` here — it's about a transformation of the code...